sound150@claudiux/files/sound150@claudiux/po/fr.po

```diff
@@ -342,47 +344,47 @@ msgstr "Contrôle du volume maximum"
 #. 3.4->settings-schema.json->stepVolume->options
 #. 2.8->settings-schema.json->stepVolume->options
 msgid "6%"
-msgstr ""
+msgstr "6%"
 
 #. 5.4->settings-schema.json->stepVolume->options
 #. 4.4->settings-schema.json->stepVolume->options
 #. 4.6->settings-schema.json->stepVolume->options
 #. 3.4->settings-schema.json->stepVolume->options
 #. 2.8->settings-schema.json->stepVolume->options
 msgid "5%"
-msgstr ""
+msgstr "±5%"
```

Comment:
Please replace, in line 355, "±5%" by "5%". Sorry.

Reply:
No stress, it's an easy fix! Hopefully this is merged soon so it'll be easier to make future translation changes.